multimodal-search-pdf/pods/ranker.yml

```diff
@@ -0,0 +1,24 @@
+jtype: WeightedRanker
+with:
+  query_required_keys:
+    - weight
+metas:
+  py_modules:
+    - weighted_ranker.py
+requests:
+  use_default: true
+  on:
+    SearchRequest:
+      - jtype: Chunk2DocRankDriver
```

Comment:
Add docstring for describing `Chunk2DocRankDriver` usage, and why we use `sortQL` and `sliceQL` filters